Modules/_datetimemodule.c

```diff
@@ -663,6 +663,144 @@ set_date_fields(PyDateTime_Date *self, int y, int m, int d)
     SET_DAY(self, d);
 }
 
+/* ---------------------------------------------------------------------------
+ * String parsing utilities and helper functions
+ */
+static inline unsigned int to_int(char ptr)
```

Comment:
I don't think "static inline" is interpreted differently from "static" by any relevant compilers.  AFAIK, we don't use "static inline" elsewhere in CPython code.  Please just use "static".

Reply:
Not a problem. I translated these from macros in my original code so it seemed like `inline` was appropriate at the time, but I don't have strong opinions in this regard.